bin/make_run_ap_pipe_conf.py

```diff
@@ -0,0 +1,24 @@
+import os
+'''Generate configuration file for ap_pipe slurm run
```

Comment:
maybe the filename should make clear this is only for HiTS?  unless you want to make visits configurable in some way.